### ISSUE TITLE
example project returned `Step implementation not found`

### ISSUE THREAD

**Issue by anucha-tdg** [BugDiver/gauge-ts#165]
**Describe the bug**
Simple project showed an error. It's happen with example project also.

**To Reproduce**
1. `gauge init ts projectname`
2. `gauge run specs` 

**Expected behavior**

**Screenshots**
![Screenshot 2566-02-28 at 19 53 45](https://user-images.githubusercontent.com/122740311/221859615-ea9ede33-f2f7-437b-8164-139ba9864cf0.png)
![Screenshot 2566-02-28 at 19 54 31](https://user-images.githubusercontent.com/122740311/221859760-5e31ef06-2a34-44c5-8e07-6055e0dbefb5.png)

**Desktop (please complete the following information):**
 - OS: Macos Intel
 - Gauge and plugin version 
 ```
 Gauge version: 1.4.3
Commit Hash: f98dd40

Plugins
-------
html-report (4.2.0)
js (2.4.0)
python (0.3.17)
screenshot (0.1.0)
ts (0.1.0)
```
 - Node version `v18.13.0`
 - Npm version `8.19.3`

**Comment by anucha-tdg** [comment]
This is example project.
<img width="1680" alt="Screenshot 2566-02-28 at 19 56 59" src="https://user-images.githubusercontent.com/122740311/221860337-ceb21538-35f4-492d-b9f9-1fd383d50820.png">

**Comment by anucha-tdg** [comment]
Now fixed by follow this #57 update version **gauge-ts to 0.1.1**
